### PR TITLE
storage: write synced transactions to synced-transactions table

### PR DESCRIPTION
**Pull request by bmwill**

### CODE REVIEW

File: crates/sui-core/src/authority/authority_store_tables.rs [hunk 1]
```diff
@@ -58,6 +58,7 @@ pub struct AuthorityPerpetualTables {
     pub(crate) executed_effects: DBMap<TransactionDigest, SignedTransactionEffects>,
 
     pub(crate) effects: DBMap<TransactionEffectsDigest, TransactionEffects>,
```
Comment (lxfind) [inline crates/sui-core/src/authority/authority_store_tables.rs:60]: Could you rename this to synced_effects?

**Comment by williampsmith** [comment]
May we also want the same for transaction effects @bmwill @mwtian ?

File: crates/sui-core/src/authority/authority_store_tables.rs (hunk 1 above)
Comment (bmwill) [reply crates/sui-core/src/authority/authority_store_tables.rs:60]: Lets do that in a follow up. Right now state sync will write to this table, as well as when effects are written to the executed_effects table so `synced_effects` 100% accurate atm either.

**Comment by mwtian** [comment]
> May we also want the same for transaction effects @bmwill @mwtian ?

Yes, it seems it will be done in another PR.

File: crates/sui-core/src/authority/authority_store_tables.rs (hunk 1 above)
Comment (mwtian) [reply crates/sui-core/src/authority/authority_store_tables.rs:60]: It seems either we can remove the write to `effects` table after execution, or we can create another table just for synced effects.

**Pull request merged**